### ISSUE TITLE
Test Helper for Schedule

### ISSUE THREAD

**Issue by skorfmann** [winglang/wing#2845]
### Feature Spec

Make it easy and less verbose to test a `Schedule` 

### Use Cases

To make the inflight handler of a schedule testable, I'll have to define it explicitly like this so that I have a reference to it.

```
let schedule = new cloud.Schedule(cron: "0 * * * ?");

let scheduleHandler = inflight () => {
  log("hello")
};

schedule.onTick(scheduleHandler);
```

Would be nicer to being able to define it like:

```
let schedule = new cloud.Schedule(cron: "0 * * * ?");

schedule.onTick(inflight () => {
  log("hello")
};
```

and invoke it with something like `schedule.tick()` or `TestHelper.tick(schedule)`. 

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by Chriscbr** [comment]
Yes, I've wondered if we should have some kind of "test" or "tick" or "trigger" or "run" or some other way to manually trigger the scheduled function. It does seem feel like apps using this resource are hard to test at the moment
